<launch>
    <arg name="dataset_folder"  default="/data/Maicity/" />
    <arg name="sequence_number" default="01" />
    <arg name="to_bag"          default="true" />
    <arg name="output_bag_file" default="$(arg dataset_folder)bag/maicity$(arg sequence_number)_norm.bag" />
    <arg name="publish_delay"   default="4" />
    <arg name="bin_name_width"  default="5" />
    <arg name="unit_fix"        default="1" />

    <node name="kittiHelperSimple" pkg="aloam_velodyne" type="kittiHelperSimple" output="screen"> 
        <param name="dataset_folder"    type="string"   value="$(arg dataset_folder)" />
        <param name="sequence_number"   type="string"   value="$(arg sequence_number)" />
        <param name="to_bag"            type="bool"     value="$(arg to_bag)" />
        <param name="output_bag_file"   type="string"   value="$(arg output_bag_file)" /> <!-- replace with your output folder -->
        <param name="publish_delay"     type="int"      value="$(arg publish_delay)" />
        <param name="bin_name_width"    type="int"      value="$(arg bin_name_width)"/>
        <param name="unit_fix"          type="int"      value="$(arg unit_fix)"/>
    </node>
</launch>